src/index.html

```diff
@@ -7,8 +7,49 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="header__wrapper">
+        <div class="header__logo">
+          <a href="#home" class="header__logo--link">
+            <img alt="Logo" class="header__logo--img"
+            src="./images/logo.png">
```

Comment:
```suggestion
            <img 
              alt="Logo" 
              class="header__logo--img"
              src="./images/logo.png"
            >
```
proofread your code with the checklist
<img width="987" alt="image" src="https://github.com/mate-academy/layout_moyo-header/assets/102411130/6626cfaf-c00a-4176-bd7f-aaffc436b6b3">